CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.7.1] - 2019-05-27
```

Comment:
```suggestion
## [Unreleased]
```

Reply:
Upon release we will decide what the version bump is based on all the changes that have been done.